pkg/smokescreen/smokescreen.go

```diff
@@ -53,6 +53,7 @@ type smokescreenContext struct {
 	decision  *aclDecision
 	proxyType string
 	logger    *logrus.Entry
+	requestedHost string
```

Comment:
This needs to be `gofmt`d, and we should be failing commits in CI which haven't. If you're interested in adding this to the Github Actions please do!